plugins/woocommerce-blocks/docs/third-party-developers/extensibility/data-store/README.md

```diff
@@ -55,15 +55,17 @@ const { SCHEMA_STORE_KEY } = window.wc.wcBlocksData;
 The **[Store Notices Store (wc/store/store-notices)](store-notices.md)** is dedicated to handling various store notices and alerts. To access this store using a constant, you can use:
 
 ```ts
-const { STORE_NOTICES_STORE_KEY } = window.wc.wcBlocksData;
+const { storeNoticesStore } = window.wc.wcBlocksData;
```

Comment:
Should we rename this to `noticesStore` instead? Bit of a mouthful otherwise

Reply:
While it is a little longer, it correlates directly with the name of the store: `wc/store/store-notices` I think I'd prefer to keep this direct reference.

Reply:
What about renaming the store to `notices` to better fit with the other store names?

Reply:
It's possible, but I don't think it's a necessary change. The purpose of this store is to work with `StoreNoticesContainer`. In my opinion the name is useful and specific.

Reply:
Also, it's unlikely that anyone is using it, but this data store is technically public API so renaming it would break that.

Reply:
Ok I'm convinced